app/controllers/v0/map_services_controller.rb

```diff
@@ -0,0 +1,40 @@
+# frozen_string_literal: true
+
+require 'map/security_token/service'
+
+module V0
+  class MapServicesController < SignIn::ApplicationController
+    skip_before_action :authenticate
+    before_action :load_service_account_config
+
+    # POST /v0/map_services/token
+    def token
+      result = MAP::SecurityToken::Service.new.token(
+        application: @service_account_config.service_account_id.to_sym,
+        icn: params[:icn]
```

Comment:
I've been thinking about this particular part, I'm thinking we should leverage the `sub` field in the `service_account_access_token`, rather than let this be a param to input. The reason is that as written, if a service account access token was stolen, someone could potentially gain MAP tokens for a collection of users, whereas if we force the service to put the ICN in the `sub` field, a stolen token only gives you access to a single user

Reply:
I separated out the work needed to retrieve an ICN from the provided access token in this PR: https://github.com/department-of-veterans-affairs/vets-api/pull/14564